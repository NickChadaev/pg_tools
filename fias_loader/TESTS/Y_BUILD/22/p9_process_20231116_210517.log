2023-11-16 21:05:17 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_22 postgres /home/n.chadaev@abrr.local/Y_BUILD/22/stage_gar_c_22.csv 2023-11-13
--------------------------------------------------------
[21:05:17] ... Алтайский край, id_region: 23, Execute: /home/n.chadaev@abrr.local/Y_BUILD/22/stage_gar_c_22.csv, process_id:25345
[21:05:17] ... Start process
[21:05:17] ...  -- Version
[21:05:18] ...  -- Очистка от данных
[21:05:18] ...  -- Убираем индексы
[21:05:18] ...  -- Set UNLOGGED
[21:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[21:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[21:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[21:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[21:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[21:05:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[21:05:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[21:05:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[21:05:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[21:05:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[21:05:19] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_REESTR_OBJECTS_20231113_fe4e9d36-11e9-4c8d-8ab5-41d02d7692de.XML
[21:26:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_ADDR_OBJ_20231113_94446c66-4780-4f5f-ac31-38a3a8b8259c.XML
[21:27:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_HOUSES_20231113_be160099-5267-416e-933b-6fdf3d51f0b7.XML
[21:44:57] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_STEADS_20231113_44c25d18-48c4-4cec-9fdf-ee751c298346.XML
[21:47:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_APARTMENTS_20231113_b8050b9d-65ad-4f24-bebe-62cfd2aa29ff.XML
[21:56:43] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_ROOMS_20231113_f76f516f-86a5-4722-a3b9-09853db6c8ad.XML
[21:56:51] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_CARPLACES_20231113_dfe53974-db7e-48bc-9d12-96ee2ef7ad15.XML
[21:56:53] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_ADM_HIERARCHY_20231113_52fcce5a-025a-4fc5-89e9-d6caa1133526.XML
[22:18:58] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_MUN_HIERARCHY_20231113_82225524-798d-4938-b277-2a80c0a27116.XML
[22:40:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_CHANGE_HISTORY_20231113_3333c0e1-ee49-4538-97af-63dd741914cd.XML
[23:43:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_ADDR_OBJ_DIVISION_20231113_64ee4cde-3aee-46eb-8356-fb7c5093a52d.XML
[23:43:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_ADDR_OBJ_PARAMS_20231113_bc2786c6-b67c-43c1-b127-9f04c829940b.XML
[23:48:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_STEADS_PARAMS_20231113_48fbdaa1-2e2a-4af9-849e-511e473960c4.XML
[00:11:25] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_HOUSES_PARAMS_20231113_bc158a2e-c1d0-4232-a235-73694d80a7dc.XML
[02:02:08] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_APARTMENTS_PARAMS_20231113_67ce40cd-2609-4ae7-82ee-3d592aa812c8.XML
[02:19:39] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_ROOMS_PARAMS_20231113_5c3bcc16-7cbd-42d6-9a21-98974eaff8eb.XML
[02:19:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_CARPLACES_PARAMS_20231113_03da5c88-3ce0-4ddc-83c2-8dace7e295d0.XML
[02:19:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/22/AS_NORMATIVE_DOCS_20231113_f4f52dea-2d67-4a21-b685-0553d9f80c9a.XML
[02:22:31] ...  -- Data finish
[02:22:31] ...  -- Set LOGGED
[02:23:14] ... Stop process
--------------------------------------------------------
