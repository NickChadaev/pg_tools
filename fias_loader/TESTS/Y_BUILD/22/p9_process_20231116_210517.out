 save_gar_version 
------------------
               82
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                82
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 1648213
gar_fias.as_addr_obj: 26178
gar_fias.as_houses: 1332307
gar_fias.as_steads: 175503
gar_fias.as_apartments: 784346
gar_fias.as_rooms: 11762
gar_fias.as_carplaces: 2694
gar_fias.as_adm_hierarchy: 1725547
gar_fias.as_mun_hierarchy: 1704472
gar_fias.as_change_history: 5118642
gar_fias.as_addr_obj_division: 69
gar_fias.as_addr_obj_params: 348405
gar_fias.as_steads_params: 1864008
gar_fias.as_houses_params: 9088863
gar_fias.as_apartments_params: 1467426
gar_fias.as_rooms_params: 12793
gar_fias.as_carplaces_params: 165
gar_fias.as_normative_docs: 231514
